rt-forecast-2/forecast/utils/run-rt-forecast.R

```diff
@@ -30,21 +30,30 @@ run_rt_forecast <- function(deaths,
     
     models <- models[!targets_present]
   }
-  
-  if (length(models) == 0) {
-    return(invisible(NULL))
-  }
 
   # Format for epinow2 ------------------------------------------------------
   
+  # Deaths data
   if (!missing(submission_date)) {
     deaths <- dplyr::filter(deaths, date < submission_date)
   }
-  
   deaths <- setDT(deaths)
   deaths <- deaths[, .SD[date >= (max(date) - lubridate::weeks(8))], by = region]
   data.table::setorder(deaths, date)
   
+  # Rt global options
+  rt <- opts_list(rt_opts(), deaths) %>%
+    purrr::transpose()
+  
+  # - Population data
+  rt$pop <- suppressMessages(readr::read_csv("utils/state_pop_totals.csv")) %>%
+    dplyr::pull(tot_pop, name = state_name) %>%
+    as.list()
+  
+  # - Prior
+  rt$prior <- purrr::map(rt$prior, ~ list(mean = 1, sd = 0.2))
```

Comment:
this is a good approach but could have been done in the first rt_opts() call at top instead